python3.8-slim-bullseye-bedtools-venv/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.8-slim-bullseye
```

Comment:
This is copied from the other Dockerfile, the only things that changed are 
- python:3.8-slim-bullseye instead of python:3.8-slim
- An updated version of bedtools